apps/backend/src/authn/authn.service.ts

```diff
@@ -28,6 +34,27 @@ export class AuthnService {
     }
   }
 
+  async validateApiKey(apikey: string): Promise<User | null> {
+    try {
+      const jwtPayload = jwt.verify(
+        apikey,
+        this.configService.get('JWT_SECRET') || 'disabled'
```

Comment:
This is passing the string literal to the key verification step so everything will fail if its disabled. But if you can check if the key is disabled, why is this function getting called & polluting logic within it ? You can raise a better error if you don't let the program get this far and raise something before this step.